packages/react/src/primitives/types/image.ts

```diff
@@ -48,6 +54,12 @@ export interface ImageOptions extends ImageStyleProps {
    */
   srcSet?: React.ImgHTMLAttributes<HTMLImageElement>['srcSet'];
 
+  /**
+   * @description
+   * Image will be loaded lazily if set to true.
```

Comment:
Should we note what the default value will be here?

Reply:
Makes sense